docs/generated/sql/functions.md

```diff
@@ -3228,6 +3228,8 @@ Note: the name pattern must contain ASCII letters already for capital letters to
 <tr><td><a name="crdb_internal.is_constraint_active"></a><code>crdb_internal.is_constraint_active(table_name: <a href="string.html">string</a>, constraint_name: <a href="string.html">string</a>) &rarr; <a href="bool.html">bool</a></code></td><td><span class="funcdesc"><p>This function is used to determine if a given constraint is currently.
 active for the current transaction.</p>
 </span></td><td>Volatile</td></tr>
+<tr><td><a name="crdb_internal.is_system_table_key"></a><code>crdb_internal.is_system_table_key(raw_key: <a href="bytes.html">bytes</a>) &rarr; <a href="bool.html">bool</a></code></td><td><span class="funcdesc"><p>This function is used only by CockroachDB’s developers for testing purposes.</p>
```

Comment:
this is an internal function, so it should not appear in the docs. use `Undocumented:     true,` on the builtin.